src/wp-includes/media.php

```diff
@@ -5317,9 +5325,9 @@ function wp_media_personal_data_exporter( $email_address, $page = 1 ) {
  */
 function _wp_add_additional_image_sizes() {
 	// 2x medium_large size.
-	add_image_size( '1536x1536', 1536, 1536 );
+	add_image_size( '1536x1536', 1536, 1536, false, true );
 	// 2x large size.
-	add_image_size( '2048x2048', 2048, 2048 );
+	add_image_size( '2048x2048', 2048, 2048, false, true );
```

Comment:
Related to my above comment on `post-thumbnail` size: We should also _only_ pass `true` there, but not make it a parameter on `set_post_thumbnail_size()`. This way it's consistent with the code here. All core sizes are opted in, only custom sizes are opted out; and if you want to opt out any core size, you'll have to use the filter.

Reply:
that makes sense, will do

Reply:
updated